lib/instrumentation/index.js

```diff
@@ -83,6 +84,17 @@ function Instrumentation (agent) {
   }
 }
 
+Object.defineProperty(Instrumentation.prototype, 'ids', {
+  get () {
+    var current = this.currentSpan || this.currentTransaction
+    return current ? current.ids : new Ids()
+  }
+})
+
+Instrumentation.prototype.toString = function () {
```

Comment:
Having a `toString()` on the `Instrumentation` object seems a little odd to me... what's the use-case for this?

Reply:
Just to align with the other places, but could probably just remove that. 🤷‍♂

Reply:
fixed